apps/picsa-tools/manual-tool/src/app/components/stepsContainer/stepsContainer.component.scss

```diff
@@ -0,0 +1,62 @@
+.mobileManualPage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: start;
+  padding: 1rem;
+
+  .simpleSection {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    .ActivityHeading {
```

Comment:
nit(non-blocking)
I'd probably avoid against deeply nested styles as it makes it harder to refactor. All angular components already scope styles within a shadow dom so just defining `.ActivityHeading` at the top level should be fine (won't interfere with any other components even if named the same classes)

Reply:
Ohh did not know that thank you